prov/verbs/src/verbs_msg_ep.c

```diff
@@ -561,30 +561,37 @@ static inline int fi_ibv_poll_reap_unsig_cq(struct fi_ibv_msg_ep *ep)
 	return FI_SUCCESS;
 }
 
+static inline ssize_t
+fi_ibv_unsignaled_send(struct fi_ibv_msg_ep *ep, struct ibv_send_wr *wr)
+{
+	int ret = FI_IBV_INVOKE_POST(send, send, ep->id->qp, wr, NULL);
+	if (OFI_UNLIKELY(ret))
+		return ret;
+
+	/* `+ 1` means to make space for the next unsignaled send. */
+	if ((ofi_atomic_inc32(&ep->scq->sends_outstanding) + 1) >=
```

Comment:
Does verbs return something as simply as EBUSY to indicate that the queue is full that we could use to poll the cq, then retry the request?

Reply:
Nice point! I've updated PR.